src/precice/config/ParticipantConfiguration.cpp

```diff
@@ -126,56 +126,46 @@ ParticipantConfiguration::ParticipantConfiguration(
   tag.addSubtag(tagWatchIntegral);
 
   XMLTag tagProvideMesh(*this, TAG_PROVIDE_MESH, XMLTag::OCCUR_ARBITRARY);
-  doc = "Provide a mesh (see tag <mesh>) to other participants.";
+  doc = "Provide a mesh (see tag `<mesh>`) to other participants.";
   tagProvideMesh.setDocumentation(doc);
   attrName.setDocumentation("Name of the mesh to provide.");
   tagProvideMesh.addAttribute(attrName);
   tag.addSubtag(tagProvideMesh);
 
   XMLTag tagReceiveMesh(*this, TAG_RECEIVE_MESH, XMLTag::OCCUR_ARBITRARY);
-  doc = "Makes a remote mesh (see tag <mesh>) available to this participant.";
+  doc = "Makes a remote mesh (see tag `<mesh>`) available to this participant.";
```

Comment:
This could also include info regarding the decomposition, "region of interest" and geometric filter.

Reply:
I think we could also mention here that the mesh is "made available by communicating it".

Reply:
Let's have a look at these again at the Coding Days.